container-images/mds-web-sockets/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "build": "tsc --build tsconfig.build.json",
     "bundle": "yarn build && webpack --mode=production --env.npm_package_name=${npm_package_name} --env.npm_package_version=${npm_package_version}",
-    "image": "yarn bundle && ../../bin/mdsctl -c:image-name=mds-compliance:${npm_package_version} buildImage",
+    "image": "yarn bundle && ../../bin/mdsctl -c:image-name=mds-web-sockets:${npm_package_version} buildImage",
```

Comment:
Just curious... what's the difference between `npm_package_name` and what we're using for `image-name` here? No interest in similarly setting this via config?